src/lib/common/NotificationFormat.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+*
+* Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Ken Zangelin
+*/
+#include <string.h>
+#include <string>
+#include <sstream>
+
+#include "logMsg/logMsg.h"
+#include "logMsg/traceLevels.h"
+#include "common/NotificationFormat.h"
+#include "common/wsStrip.h"
+
+
+
+/* ****************************************************************************
+*
+* notificationFormatToString - 
+*/
+const char* notificationFormatToString(NotificationFormat format, bool noDefault)
```

Comment:
I will just make a quick change in the code and compile. 
Sometimes empirical methods are faster :-)

That said, personally I don't agree in using std::string as default type, instead of char*.
Unless 'needed' (to avoid using strcmp instead of '=='), it only adds overhead. We gain nothing and the broker gets less efficient. No, instead **char*** should be the default type to use and std::string only in the exceptions where we want to do the comparisons.

That is only my personal opinion, of course.